feg/cloud/go/services/health/servicers/southbound/health.go

```diff
@@ -36,7 +36,7 @@ import (
 type HealthStatus int
 
 type HealthServer struct {
-	store storage.HealthBlobstore
+	Store storage.HealthBlobstore
```

Comment:
changing to public for testing, i.e. in feg/cloud/go/services/health/servicers/health_test_service.go
Ig I could rework the tests too, lmk preference

Reply:
Normally I'd say let's rework the tests, but for sake of expediency this is fine